CRITICAL - 2024-06-28 16:00:15 --> ErrorException: Undefined array key 0
in APPPATH\Controllers\Admin\Cotizaciones.php on line 104.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 104)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 16:00:20 --> ErrorException: Undefined array key 0
in APPPATH\Controllers\Admin\Cotizaciones.php on line 104.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 104)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:24:30 --> ErrorException: Undefined variable $request
in APPPATH\Controllers\Admin\Cotizaciones.php on line 142.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(142): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $request', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 142)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 17:26:01 --> mysqli_sql_exception: Field 'id_cotizacion' doesn't have a default value in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `se...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\sellopronto\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 17:26:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:26:01 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:26:01 --> [Caused by] mysqli_sql_exception: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 17:32:58 --> mysqli_sql_exception: Field 'id_cotizacion' doesn't have a default value in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `se...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\sellopronto\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 17:32:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:32:58 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:32:58 --> [Caused by] mysqli_sql_exception: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Admin\Cotizaciones.php(148): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 17:37:21 --> mysqli_sql_exception: Field 'id_cotizacion' doesn't have a default value in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `se...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\sellopronto\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 17:37:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:37:21 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:37:21 --> [Caused by] mysqli_sql_exception: Field 'id_cotizacion' doesn't have a default value
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 17:42:25 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE) in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `se...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\sellopronto\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 17:42:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:42:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:42:25 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 17:43:02 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE) in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `se...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\sellopronto\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 17:43:02 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:43:02 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:43:02 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 17:47:23 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE) in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `se...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `se...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `se...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `se...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\sellopronto\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert(Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 17:47:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 2 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:47:23 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 4 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 17:47:23 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`sellopro`.`sellopro_detalles`, CONSTRAINT `FK_sellopro_detalles_sellopro_cotizaciones` FOREIGN KEY (`id_cotizacion`) REFERENCES `sellopro_cotizaciones` (`idQt`) ON DELETE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `sellopro_detalles` (`id_articulo`) VALUES (:id_articulo:)', [...], false)
 5 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Admin\Cotizaciones.php(149): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 18:23:13 --> ErrorException: Undefined array key 0
in APPPATH\Controllers\Admin\Cotizaciones.php on line 104.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(104): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 104)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 18:38:41 --> ErrorException: Undefined variable $db
in APPPATH\Controllers\Admin\Cotizaciones.php on line 152.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(152): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $db', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 152)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 18:39:27 --> ErrorException: Undefined variable $cotizacion
in APPPATH\Controllers\Admin\Cotizaciones.php on line 154.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(154): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cotizacion', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 154)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 18:40:33 --> ErrorException: Undefined variable $cotizacion
in APPPATH\Controllers\Admin\Cotizaciones.php on line 154.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(154): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cotizacion', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 154)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 18:41:33 --> ErrorException: Undefined variable $cotizacion
in APPPATH\Controllers\Admin\Cotizaciones.php on line 154.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(154): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cotizacion', 'C:\\laragon\\www\\sellopronto\\app\\Controllers\\Admin\\Cotizaciones.php', 154)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar_ind()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 19:30:07 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 217.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(217): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('1')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 19:51:29 --> mysqli_sql_exception: Incorrect integer value: '' for column 'descuento' at row 1 in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `sellopr...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sellopr...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sellopr...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopr...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(987): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\laragon\www\sellopronto\system\Model.php(807): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 19:51:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Incorrect integer value: '' for column 'descuento' at row 1
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopro_clientes` SET `nombre` = :nombre:, `correo` = :correo:, `direccion` = :direccion:, `telefono` = :telefono:, `ciudad` = :ciudad:, `estado` = :estado:, `descuento` = :descuento:
WHERE `sellopro_clientes`.`idCliente` IN :sellopro_clientes.idCliente:', [...], false)
 2 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 19:51:29 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Incorrect integer value: '' for column 'descuento' at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopro_clientes` SET `nombre` = :nombre:, `correo` = :correo:, `direccion` = :direccion:, `telefono` = :telefono:, `ciudad` = :ciudad:, `estado` = :estado:, `descuento` = :descuento:
WHERE `sellopro_clientes`.`idCliente` IN :sellopro_clientes.idCliente:', [...], false)
 4 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', [...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 19:51:29 --> [Caused by] mysqli_sql_exception: Incorrect integer value: '' for column 'descuento' at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopro_clientes` SET `nombre` = :nombre:, `correo` = :correo:, `direccion` = :direccion:, `telefono` = :telefono:, `ciudad` = :ciudad:, `estado` = :estado:, `descuento` = :descuento:
WHERE `sellopro_clientes`.`idCliente` IN :sellopro_clientes.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-06-28 19:52:14 --> mysqli_sql_exception: Incorrect integer value: '' for column 'descuento' at row 1 in C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php:306
Stack trace:
#0 C:\laragon\www\sellopronto\system\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `sellopr...', 0)
#1 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sellopr...')
#2 C:\laragon\www\sellopronto\system\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sellopr...')
#3 C:\laragon\www\sellopronto\system\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopr...', Array, false)
#4 C:\laragon\www\sellopronto\system\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
#5 C:\laragon\www\sellopronto\system\BaseModel.php(987): CodeIgniter\Model->doUpdate(Array, Array)
#6 C:\laragon\www\sellopronto\system\Model.php(807): CodeIgniter\BaseModel->update(Array, Array)
#7 C:\laragon\www\sellopronto\app\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', Array)
#8 C:\laragon\www\sellopronto\system\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
#9 C:\laragon\www\sellopronto\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
#10 C:\laragon\www\sellopronto\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\laragon\www\sellopronto\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-06-28 19:52:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Incorrect integer value: '' for column 'descuento' at row 1
in SYSTEMPATH\Database\BaseConnection.php on line 647.
 1 SYSTEMPATH\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopro_clientes` SET `nombre` = :nombre:, `correo` = :correo:, `direccion` = :direccion:, `telefono` = :telefono:, `ciudad` = :ciudad:, `estado` = :estado:, `descuento` = :descuento:
WHERE `sellopro_clientes`.`idCliente` IN :sellopro_clientes.idCliente:', [...], false)
 2 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 3 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 4 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 5 APPPATH\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', [...])
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 19:52:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Incorrect integer value: '' for column 'descuento' at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 311.
 1 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 2 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopro_clientes` SET `nombre` = :nombre:, `correo` = :correo:, `direccion` = :direccion:, `telefono` = :telefono:, `ciudad` = :ciudad:, `estado` = :estado:, `descuento` = :descuento:
WHERE `sellopro_clientes`.`idCliente` IN :sellopro_clientes.idCliente:', [...], false)
 4 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 5 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 6 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 7 APPPATH\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', [...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 19:52:14 --> [Caused by] mysqli_sql_exception: Incorrect integer value: '' for column 'descuento' at row 1
in SYSTEMPATH\Database\MySQLi\Connection.php on line 306.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(306): mysqli->query('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(693): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 3 SYSTEMPATH\Database\BaseConnection.php(607): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `sellopro_clientes` SET `nombre` = \'Fabian Reyes\', `correo` = \'\', `direccion` = \'\', `telefono` = \'55 4982 3829\', `ciudad` = \'\', `estado` = \'Choose...\', `descuento` = \'\'
WHERE `sellopro_clientes`.`idCliente` IN (\'16\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2474): CodeIgniter\Database\BaseConnection->query('UPDATE `sellopro_clientes` SET `nombre` = :nombre:, `correo` = :correo:, `direccion` = :direccion:, `telefono` = :telefono:, `ciudad` = :ciudad:, `estado` = :estado:, `descuento` = :descuento:
WHERE `sellopro_clientes`.`idCliente` IN :sellopro_clientes.idCliente:', [...], false)
 5 SYSTEMPATH\Model.php(393): CodeIgniter\Database\BaseBuilder->update()
 6 SYSTEMPATH\BaseModel.php(987): CodeIgniter\Model->doUpdate([...], [...])
 7 SYSTEMPATH\Model.php(807): CodeIgniter\BaseModel->update([...], [...])
 8 APPPATH\Controllers\Admin\Clientes.php(52): CodeIgniter\Model->update('16', [...])
 9 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Clientes->actualizar()
10 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Clientes))
11 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:08:14 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 220.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(220): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:08:46 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 220.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(220): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:09:58 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 220.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(220): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:10:08 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 220.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(220): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:11:53 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 222.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(222): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:27:43 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 222.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(222): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:30:18 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:30:53 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:32:06 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:36:45 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:37:45 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('2')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 20:38:37 --> TypeError: Unsupported operand types: string * string
in APPPATH\Controllers\Admin\Cotizaciones.php on line 126.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:40:03 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('3')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:40:13 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('3')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-06-28 20:40:17 --> TypeError: Unsupported operand types: string * string
in APPPATH\Controllers\Admin\Cotizaciones.php on line 126.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->agregar()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:42:12 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('3')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
WARNING - 2024-06-28 20:42:15 --> [DEPRECATED] number_format(): Passing null to parameter #1 ($num) of type float is deprecated in APPPATH\Controllers\Admin\Cotizaciones.php on line 221.
 1 APPPATH\Controllers\Admin\Cotizaciones.php(221): number_format(null, 2)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Admin\Cotizaciones->mostrar_detalles('3')
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin\Cotizaciones))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
